src/concepts/history.md

```diff
@@ -30,12 +30,11 @@ Daily Users, which was [renamed to Active Daily
 Installs](https://bugzilla.mozilla.org/show_bug.cgi?id=812282) (ADI) by
 2012.
 
-Work is underway to replace `blocklist.xml` with a [Remote
+As of August 2020, this mechanism has been superseded by a [Remote
```

Comment:
Could change this to Fx72, which is when the staged rollout of the remote settings-based implementation started, or Fx73 when the new implementation became the default.

Reply:
Sounds good, I think just referencing 73 is sufficient context.